### PR TITLE
snap icons/cursors

### PR DESCRIPTION
**Pull request by rduivenvoorde**
Playing with the snapping tools, and thinking: "Mmm there should be a different icon when it snaps to a intersection", I suddenly realized that there WAS an difference, although (to me) not VERY clear, the snapping to a segment is an hourglass (filled X) while to a intersection is a plain X.

Although I would prefer to ask the list to change one of them to a circle or so (to better distinguish the two) I thought: let's put it in the docs.

I created some images (could be made smaller) and wrote some text. I'm not sure if we should call these icons or cursors or pointers... 

So: happy to discuss this and change it to whatever works best.

### CODE REVIEW

**Comment by rduivenvoorde** [comment]
With me it looks like this: feel free to provide better idea's:

Screenshot-20200610131128-825x843

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -124,6 +124,28 @@ snapping enabled layers, even if there are no vertices at the intersections.
...
+   Snapping on a segment: hourglass icon
+
+
+.. figure:: img/snap_intersection_icon.png
+
+   Snapping on an intersection: cross icon
+
+
+Note that it is possible to change the change the color of these icons in the 
+'Digitizing' part of your settings.
```
[the remark this answers is not in the document]
Comment (rduivenvoorde) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Mmm, this url to objects.inv should have worked... I fixed that at around 14:00 CET yesterday... See if Travis is ok now...

**Comment by rduivenvoorde** [comment]
Are the images not too large? Takes up a lot of page space now... Happy to clip them.

**Comment by havatv** [comment]
> Are the images not too large? Takes up a lot of page space now... Happy to clip them.

Feel free to clip/crop them! :+1:

**Comment by DelazJ** [comment]
How about having them all in a single row? Either within a table, or horizontally merged in a single image. Less scroll generated by them and they will be less bigger, since they will share the page width.

**Comment by ghtmtt** [comment]
ref #2209

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -122,8 +122,31 @@ Another available option is to use |snappingIntersection| :guilabel:`snapping on
...
+
+   Snapping to a vertex: box icon
+
+.. figure:: img/snap_segment_icon.png
+
+   Snapping to a segment: hourglass icon
+
+.. figure:: img/snap_intersection_icon.png
+
+   Snapping to an intersection: cross icon
```
Comment (havatv) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:142]:
```suggestion
.. figure:: img/snap_examples.png

   Snapping. To a vertex: box icon (left); to a segment: hourglass icon (middle), to an intersection: cross icon (right)
```
And add the merged image as "snap_examples.png".

**Pull request merged**